### PR TITLE
use window size, update on window resize

### PR DESCRIPTION
**Pull request by xdamman**
Follow up to https://github.com/opencollective/opencollective-frontend/pull/950
based on @znarf's feedback

### CODE REVIEW

File: src/components/GoalsCover.js [hunk 1]
```diff
@@ -62,8 +62,8 @@ class GoalsCover extends React.Component {
   }
 
   componentDidMount() {
-    const state = this.populateGoals();
-    this.setState(state);
+    this.populateGoals();
+    window.onresize = () => this.populateGoals();
```
Comment (znarf) [inline src/components/GoalsCover.js]:
```suggestion
    window.addEventListener('resize', this.populateGoals);
```

Comment (znarf) [reply src/components/GoalsCover.js]: Also, maybe you want to throttle that:

```suggestion
    window.addEventListener('resize', throttle(this.populateGoals.bind(this), 400));
```

Comment (xdamman) [reply src/components/GoalsCover.js]: good idea. Fix coming (need to add `throttle` from `lodash`)

**Comment by Betree** [comment]
Implemented in https://github.com/opencollective/opencollective-frontend/pull/1214

**Pull request closed without merging**